Tests/SonarQube.TeamBuild.PreProcessor.Tests/TargetsInstallerTests.cs

```diff
@@ -74,7 +74,8 @@ public void InstallTargetsFile_Overwrite()
             CreateDummySourceTargetsFile(sourceTargetsContent1);
 
             InstallTargetsFileAndAssert(sourceTargetsContent1, expectCopy: true);
-            Assert.IsTrue(FileConstants.ImportBeforeDestinationDirectoryPaths.Count == 3, "Expecting three destination directories");
+            // TODO: this test would fail on non-Windows OSes
```

Comment:
TODO not really clear. Is it something you forgot to fix or is it something we will need to fix later on? If second case, please edit the message to give more information.

Reply:
It's something to be addressed. I'll make it clear in the comment.